### PR TITLE
Created note deletion page UI.

### LINKED ISSUE #3: Create note deletion page UI.

### PR DESCRIPTION
**Pull request by sunnyshahabuddin**
@yedhink Please Review.

### CODE REVIEW

File: app/javascript/src/components/Dashboard/Notes/DeleteAlert.jsx [hunk 1]
```diff
@@ -4,20 +4,15 @@ import { Alert } from "neetoui";
...
-}) => {
+const DeleteAlert = ({ refetch, onClose, selectedNote, setSelectedNote }) => {
   const [deleting, setDeleting] = useState(false);
-
+  const { id, title } = selectedNote;
   const handleDelete = async () => {
     try {
       setDeleting(true);
-      await notesApi.destroy({ ids: selectedNoteIds });
+      await notesApi.destroy({ ids: id });
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/DeleteAlert.jsx:13]: The keys says `ids`, that is plural. But you are sending a string. Shouldn't you be sending an array containing a single `id`?

**Review by yedhink — COMMENTED** [review]
@sunnyshahabuddin

File: app/javascript/src/components/Dashboard/Notes/DeleteAlert.jsx (hunk 1 above)
Comment (sunnyshahabuddin) [reply app/javascript/src/components/Dashboard/Notes/DeleteAlert.jsx:13]: Sorry @yedhink , I should have sent an array containing a single id. Should I send the as an array directly as `await notesApi.destroy({ ids: [id] })`  ?

Comment (yedhink) [reply app/javascript/src/components/Dashboard/Notes/DeleteAlert.jsx:13]: >await notesApi.destroy({ ids: [id] })

This is good enough.

**Pull request merged**